### PR TITLE
Remove six

### LINKED ISSUE #35: Missing declared dependency on `six` needs resolution
I understand that it's probably still desirable to support py2 in this project, but as some projects move to py3 only its becoming possible to have a package which doesn't depend on `six` 😲 !

Anyway, I think `autodocsumm` should either

A) add `six` to its dependencies (no version, I think)
or
B) treat `six` being absent as "automatic PY3 mode"

I haven't looked at the code yet, but my inclination would be to add

```python
# _compat.py

try:
    import six
except ImportError:
    PY3 = True
else:
    PY3 = six.PY3
```

and see what else needs wrapping. I can submit a PR if that would be interesting.

### PR DESCRIPTION
**Pull request by Chilipp**
actually we already declared autodocsumm to need python>=3.5 at least, so this is something that has to be removed anyway.

closes #35, ping @sirosen

### CODE REVIEW

File: autodocsumm/__init__.py [hunk 1]
```diff
@@ -188,7 +184,7 @@ def get_grouped_documenters(self, all_members=False):
         registry = get_documenters(self.env.app)
 
         for (mname, member, isattr) in self.filter_members(members, want_all):
-            classes = [cls for cls in six.itervalues(registry)
+            classes = [cls for cls in registry.values():
```
Comment (sirosen) [inline autodocsumm/__init__.py:187]: Typo, missing a `]`

**Review by sirosen — COMMENTED** [review]
Thanks for the ping. I noticed a typo.

If you want, I can try testing this version against the `marshmallow` docs (where I ran into the `six` issue) tomorrow.

File: autodocsumm/__init__.py (hunk 1 above)
Comment (Chilipp) [reply autodocsumm/__init__.py:187]: aah! thanks :see_no_evil: 
```suggestion
            classes = [cls for cls in registry.values()]:
```

**Comment by Chilipp** [comment]
> If you want, I can try testing this version against the marshmallow docs (where I ran into the six issue) tomorrow.

yes, sure. That would be helpful, thanks

**Comment by sirosen** [comment]
Just FYI, I just reran my doc build with the current `master` installed via `dependency_links=[...]` and it worked perfectly.
And I see you've been doing some CI/testing work too. Thanks for all your efforts!

**Comment by Chilipp** [comment]
Great! Thanks for the confirmation @sirosen !

The new release with this fix is available on pypi and conda-forge: 0.2.2

**Pull request merged**